### PR TITLE
fixed existing models, added team model with index assignments

### PR DESCRIPTION
**Pull request by maayazed**
Team model has no foreign keys. Both player and coach have a foreign key for team now. Added 3 new assignments to index.

### CODE REVIEW

File: models/index.js [hunk 1]
```diff
@@ -1,28 +1,41 @@
...
 });
 
+Team.hasMany(Player, {
+    foreignKey: 'team_id',
+});
+
+Team.hasOne(Coach, {
+    foreignKey: 'team_id',
+});
+
```
Comment (norgard7) [inline models/index.js:38]: Need to add
Coach.belongsTo(Team, {foreignKey: 'team_id'});

**Review by norgard7 — CHANGES_REQUESTED** [review]
Look at my comment after lin 38 in the models/index.js  We need to add one more relationship.

**Comment by arielo5** [comment]
I can add that relationship to the index model while Im working on the routes

**Comment by maayazed** [comment]
I added Coach.belongsTo(Team, {foreignKey: 'team_id'});.

Will close and reopen with new push.

**Pull request closed without merging**